presentation/src/main/java/com/pocs/presentation/model/post/PostEditUiState.kt

```diff
@@ -9,6 +9,7 @@ data class PostEditUiState(
     override val content: TextFieldValue,
     override val category: PostCategory,
     override val isUserAdmin: Boolean,
+    override val onlyMember: Boolean = true,
```

Comment:
편집 UiState 객체에는 이전 값이 필수로 들어가야하기 때문에 기본 값을 부여하지 않는게 이전 값을 전달하지 않는 위험을 줄여줍니다.
```suggestion
    override val onlyMember: Boolean,
```